src/bunit.core/ComponentParameterCollectionBuilder.cs

```diff
@@ -375,6 +377,9 @@ private static bool HasChildContentParameter()
 		=> TComponentType.GetProperty(ChildContent, BindingFlags.Public | BindingFlags.Instance) is PropertyInfo ccProp
 			&& ccProp.GetCustomAttribute<ParameterAttribute>(inherit: false) is not null;
 
+	private static bool HasGenericChildContentParameter()
+		=> TComponentType.GetProperty(ChildContent, BindingFlags.Public | BindingFlags.Instance).PropertyType.IsGenericType;
```

Comment:
Is there a chance that the `GetProperty` call can return null in this case? If yes, then we should do as in `HasChildContentParameter()`, e.g.:

```suggestion
	private static bool HasGenericChildContentParameter()
		=> TComponentType.GetProperty(ChildContent, BindingFlags.Public | BindingFlags.Instance)  is PropertyInfo ccProp
			&& ccProp.PropertyType.IsGenericType;
```

Reply:
As for that specific call, It would never return null since the `HasChildContentParameter` is called right before the call to `HasGenericChildContentParameter`. Nevertheless, what you're proposing is a safer approach. I've pushed the changes you suggested.